features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -363,7 +363,7 @@
         <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.mgt.IdentityMgtEventListener"
                        orderId="50" enable="false"/>
         <EventListener type="org.wso2.carbon.user.core.listener.UserOperationEventListener" name="org.wso2.carbon.identity.oauth.listener.IdentityOathEventListener"
-                       orderId="60" enable="false"/>
+                       orderId="60" enable="true"/>
```

Comment:
Lets remove IdentityOathEventListener and IdentityOpenIDUserEventListener from the file and add SCIMUserOperationListener to the file with enable=true